src/components/OAuthLoginButton.test.tsx

```diff
@@ -22,15 +22,15 @@ test('renders', () => {
   const screen = render(
     <OAuthLoginButton onSuccess={onSuccess} onFail={onFail} label="login" />,
   );
-  expect(screen.getByTestId('oauth-login-button')).toBeDefined();
+  expect(screen.getByTestId('button')).toBeDefined();
```

Comment:
Paper has default `testID` values.

Reply:
🤔 gut reaction: how do broad testIDs like `button` work with multiple buttons on screen?

Reply:
That is just the default, but in a more complicated test, you probably want to add your own:

```TSX
const screen = render(<OAuthLoginButton testID="button-47" />)

expect(screen.getByTestId('button-47')).toBeDefined();
```

Reply:
I'm happy to add it back though if we want a different "default" for our OAuth buttons